cylc/flow/tests/tui/test_util.py

```diff
@@ -217,14 +204,20 @@ def test_compute_tree():
                 'parents': [],
                 'cyclePoint': '1',
                 'jobs': [
-                    # mess up the order just for fun
                     {'id': 'job3', 'submitNum': '3'},
-                    {'id': 'job1', 'submitNum': '1'},
-                    {'id': 'job2', 'submitNum': '2'}
+                    {'id': 'job2', 'submitNum': '2'},
+                    {'id': 'job1', 'submitNum': '1'}
                 ]
             }
         ]
     })
+    """It computes a tree in the right structure for urwid.
+
+    Note this test doesn't use full data or propper ids because it's
+    purpose is not to test the GraphQL interface but to ensure the
+    assumptions made by compute_tree check out.
+
+    """
```

Comment:
Why has this docstring moved?

Reply:
Argh, accidental move, sorry. Will fix it tomorrow morning :+1: